packages/components/tests/dummy/app/templates/application.hbs

```diff
@@ -20,14 +20,43 @@
     {{outlet}}
   </main>
   <footer class="dummy-footer">
-    <p class="dummy-paragraph">
-      <a
-        class="dummy-link"
-        href="https://github.com/hashicorp/design-system/tree/main/packages/components"
-        target="_blank"
-        rel="noopener noreferrer"
-      >GitHub repo
-      </a>
-    </p>
+    <div class="dummy-row-wrapper">
+      <div class="dummy-row">
+        <p class="dummy-paragraph-title">Our Accessibility Statement</p>
+        <p class="dummy-paragraph">
+          Each of our components intends to conform to WCAG 2.1 AA standards (at a minimum). If you find an issue with
+          any component, you can let us know in any of the following ways:
+        </p>
+        <ol class="dummy-list">
+          <li>
+            Post a message in the #team-design-systems channel in our internal Slack
+          </li>
+          <li>
+            Email
+            <a href="mailto:design-systems@hashicorp.com">
+              design-systems@hashicorp.com
+            </a>
+          </li>
+          <li>
+            File an issue on this
+            <a href="https://github.com/hashicorp/design-system" target="_blank" rel="noopener noreferrer">GitHub
+              Repository</a>
```

Comment:
```suggestion
              repo</a>
```

non-blocking suggestion: Use same wording and casing as "GitHub repo" in the footer, and the writing guidelines under content on the scrappy site